tests/validators/tests.py

```diff
@@ -635,8 +635,8 @@
     (validate_domain_name, "python-python.com", None),
     (validate_domain_name, "python.name.uk", None),
     (validate_domain_name, "python.tips", None),
-    (validate_domain_name, "http://例子.测试", None),
-    (validate_domain_name, "http://dashinpunytld.xn---c", None),
```

Comment:
In implementing this, I found these two pre-existing test cases that I believe were erroneously passing due to the bug fixed in this PR. The domain name regex has no support for protocol. But this would be a breaking change to anyone relying on this bug.

Should we optionally support including a protocol here? My hunch is no, because it's not a valid domain name (despite obviously being a valid URL). But I wanted to raise it here in case reviewers disagree.

Reply:
Good catch, I think the proposed changes to tests look good.